src/ensembl/src/content/app/entity-viewer/shared/components/entity-viewer-sidebar/entity-viewer-sidebar-toolstrip/EntityViewerSidebarToolstrip.tsx

```diff
@@ -66,10 +66,9 @@ export const EntityViewerSidebarToolstrip = () => {
   return (
     <>
       <ImageButton
-        status={Status.DISABLED}
         description="Search"
         className={styles.sidebarIcon}
```

Comment:
did you miss to highlight the icon when enabled? minor though

Reply:
Yep, thanks for reminding

Reply:
Should be done now.